crates/iota/tests/snapshots/shell_tests__new_tests__new_files.sh.snap

```diff
@@ -0,0 +1,32 @@
+---
+source: crates/iota/tests/shell_tests.rs
+description: tests/shell_tests/new_tests/new_files.sh
+---
+----- script -----
+# Copyright (c) Mysten Labs, Inc.
+# Modifications Copyright (c) 2025 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+# basic test that iota move new outputs correct files
+iota move new example
+echo ==== files in project ====
+ls -A example
+echo ==== files in sources ====
+ls -A example/sources
+echo ==== files in tests =====
+ls -A example/tests
+
+----- results -----
+success: true
+exit_code: 0
+----- stdout -----
+==== files in project ====
+Move.toml
```

Comment:
no .gitignore but I guess it's fine

Reply:
yeah I chalk it up to differences upstream